### PR TITLE
chore: adds e2e tests for email factor on OIE

### PR DESCRIPTION
**Pull request by nikhilvenkatraman-okta**
Screen Shot 2019-11-14 at 3 09 15 PM

Resolves: OKTA-253716

### CODE REVIEW

File: test/testcafe/spec/ChallengeFactorEmail_OTP_spec.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  await identityPage.navigateToPage();
+  await identityPage.fillIdentifierField('Challenge Email');
+  await identityPage.clickNextButton();
+  return new ChallengeFactorPageObject(t);
+}
+
+test
+  .requestHooks(validOTPmock)
+  (`challenge email factor with valid OTP form has right labels`, async t => {
+    const challengeFactorPageObject = await setup(t);
```
Comment (nikhilvenkatraman-okta) [inline test/testcafe/spec/ChallengeFactorEmail_OTP_spec.js]: One thing to note while writing these tests here is that for a long time I had the assertions for pageTitle within the same test as `challenge email factor with valid OTP`.  That added a bunch of await statements in the test which started causing flakeyness. Thats why i decided to have a test just to verify labels with few awaits.

File: playground/mocks/idp/idx/data/magiclink-expired.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (haishengwu-okta) [inline playground/mocks/idp/idx/data/magiclink-expired.json]: we already have a few `terminal-return-*.json`.
are you able to reconcile ?

File: test/testcafe/spec/TerminalReturnEmail_spec.js [hunk 3]
```diff
@@ -1,4 +1,3 @@
-import IdentityPageObject from '../framework/page-objects/IdentityPageObject';
```
Comment (haishengwu-okta) [inline test/testcafe/spec/TerminalReturnEmail_spec.js]: looks like eslint doesn't catch it. mind want to follow up?

Comment (nikhilvenkatraman-okta) [reply test/testcafe/spec/TerminalReturnEmail_spec.js]: yeah, I just found this. Will create JIRA

**Pull request merged**